src/dataLoader/savedItemsDataLoader.ts

```diff
@@ -13,13 +13,9 @@ export async function batchGetSavedItemsByUrls(
   savedItemService: SavedItemDataService,
   urls: string[]
 ): Promise<SavedItem[]> {
-  let savedItems: SavedItem[] =
+  const savedItems: SavedItem[] =
     await savedItemService.batchGetSavedItemsByGivenUrls(urls);
 
-  savedItems = savedItems.filter((item) => {
-    return item.status != 'DELETED';
-  });
-
```

Comment:
just checking - will this impact non-mobile clients in a negative way? not sure how widely this change needs to be checked/circulated.

Reply:
No, since web should be (and is) using filters to retrieve saves associated with specific views (e.g. My List, Archive)